hyrisecockpit/api/app/sql/model.py

```diff
@@ -0,0 +1,44 @@
+"""Models for SQL module."""
+from typing import List
+
+from .interface import SqlQueryInterface, SqlResponseInterface
+
+
+class SqlQuery:
+    """Model of a sql query."""
+
+    def __init__(self, id: str, query: str):
+        """Initialize a sql query model."""
+        self.id: str = id
+        self.query: str = query
+
+    def update(self, interface: SqlQueryInterface) -> "SqlQuery":
+        """Update attributes of a sql query model."""
+        for key, value in interface.items():
+            setattr(self, key, value)
+        return self
+
+
+class SqlResponse:
+    """Model of a sql query response."""
+
+    def __init__(
+        self,
+        id: str,
```

Comment:
It would make sense to me to send the executed query along with the response (otherwise, the frontend would have to cache that information):

```suggestion
class SqlResponse(SqlQuery):
    """Model of a sql query response."""

    def __init__(
        self,
        id: str,
        query: str,
```

Reply:
At the moment I don't see the benefits to return the query as well. It would just overload the response. The frontend needs to wait for the reply either way (So they can easily hold on on the query). I think it should be their responsibility to keep track of the query.